### ISSUE TITLE
Show() is  not working in android.

### ISSUE THREAD

**Issue by priyangamani** [davecoffin/nativescript-filterable-listpicker#9]
@davecoffin ,show function is not working in android,I tried to use the visblility also it won't work.
var  myfilter =frame.topmost().getViewById('myfilter').visbility = "visible";//android
var frame.topmost().getViewById('myfilter').show();//ios

![image](https://user-images.githubusercontent.com/6038069/35080990-3fd5ce12-fc4c-11e7-8c21-8a191bc8ea6f.png)

**Comment by LorenDorez** [comment]
@priyangamani  you need to set visibility after you call show until @davecoffin  updates the plugin. like this

```
viewModel.page.getViewById("ListFilter").show();
                    viewModel.page.getViewById("ListFilter").visibility = "visible";
```

**Comment by davecoffin** [comment]
Alright, this is fixed 1.0.5.
